lldb/tools/debugserver/source/debugserver.cpp

```diff
@@ -156,6 +156,7 @@ RNBRunLoopMode RNBRunLoopGetStartModeFromRemote(RNBRemote *remote) {
   return eRNBRunLoopModeExit;
 }
 
+#if defined(WITH_FBS) || defined(WITH_BKS) || defined(WITH_SPRINGBOARD)
```

Comment:
Jason should comment on this one.

Reply:
Ah sorry missed this patch review.  This is fine.  Yeah we need to keep the use of is_dot_app, but putting it as a lambda used like this is perfectly fine.